pkg/domain/infosync/info_test.go

```diff
@@ -279,43 +279,3 @@ func TestTiFlashManager(t *testing.T) {
 
 	CloseTiFlashManager(ctx)
 }
-
-func TestRuleOp(t *testing.T) {
```

Comment:
Remove due to https://github.com/tikv/pd/pull/7462/files#diff-55223b29b1e0d97b9824b2bdf24c6f4a1b2b54bf408bb02182c06ffca8b8d0c3R52.